lib/routes/api/branches.js

```diff
@@ -21,11 +29,19 @@ var root = router.route('/');
     })
   });
 
-  /*
-   * PUT /:org/:repo/branches/
+  /**
+   * @api {put} /:org/:repo/branches Reorder Branches
+   * @apiUse ProjectReference
+   * @apiPermission ProjectAdmin
+   * @apiDescription Updates the branch order for a project.
+   * @apiName ReorderBranches
+   * @apiGroup Branch
+   * @apiVersion 1.0.0
    *
-   * Used to update the order of branches
+   * @apiExample {curl} CURL Example:
+   *    curl -X PUT -d branches=master,testing http://localhost/api/strider/strider-cd/branches
    *
+   * @apiParam (RequestBody) {String} branches The new branch order, comma delimited
```

Comment:
How come this param is different from the one in the doc above, regarding the request body?